fe/fe-core/src/main/java/com/starrocks/catalog/JDBCResource.java

```diff
@@ -0,0 +1,83 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.catalog;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.Lists;
+import com.google.gson.annotations.SerializedName;
+import com.starrocks.common.DdlException;
+import com.starrocks.common.proc.BaseProcResult;
+
+import java.util.Map;
+
+/*
+* External JDBC resource for JDBC table query
+* <p>
+* Example:
+* CREATE EXTERNAL RESOURCE "jdbc_pg"
+* PROPERTIES
+* (
+* "type" = "jdbc",
+* "user" = "user",
+* "password" = "password",
+* "hosts" = "127.0.0.1:1234,127.0.0.2:1235",
```

Comment:
what's the meaning of multiple hosts ?

Reply:
@mofeiatwork jdbc support multiple hosts in connection url (https://jdbc.postgresql.org/documentation/head/connect.html).

`hosts`will be deprecated and replaced by `jdbc_url`, I will update this PR later.